worker.py

```diff
@@ -890,6 +899,57 @@ def bag_full(self):
                     self.logger.warning("Failed to apply incubator {} on {}, code: {}".format(
                         inc.get('id', 0), egg.get('id', 0), ret))
 
+    async def handle_captcha(self, responses):
+        if not config.CAPTCHA_KEY:
+            self.logger.error("CAPTCHA Key is not set! this should not happen")
+            raise CaptchaException
```

Comment:
As far as I can tell it's impossible for this to be called without a `CAPTCHA_KEY`, so this test should be removed.